cmd/gdsutil/main.go

```diff
@@ -359,6 +364,23 @@ func main() {
 				},
 			},
 		},
+		{
+			Name:     "admin:tokenkey",
+			Usage:    "generate an RSA token key pair and ksuid for JWT token signing",
+			Category: "admin",
+			Action:   generateTokenKey,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "o, out",
+					Usage: "path to write keys out to (optional, will be saved as ksuid.pem by default)",
```

Comment:
if the outfile provided is the same file as the one to which the older keys were written, does this append a new key or overwrite the contents of the existing file?